content/docs/ops/setup/istioctl/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: istioctl auto-completion
+description: Istio includes a helpful script that enables auto-completion for the currently available istioctl commands.
+weight: 20
+keywords: [istioctl,bash,zsh,shell,command-line]
+---
+
+`istioctl` includes two helpful scripts that enable auto-completion for Bash and ZSH. Both of these scripts provide support the currently available `istioctl` commands.
+
+### Prerequisites
+
+{{< tabset cookie-name="prereqs" >}}
+
+{{< tab name="macOS" cookie-value="macos" >}}
+
+If you are using the macOS operating system with the Bash terminal shell, make sure that the `bash-completion` package is installed. With the [brew](https://brew.sh) package manager for macOS, you can check to see if the `bash-completion` package is installed with the following command:
+
+{{< text bash >}}
+$ brew info bash-completion
+bash-completion: stable 1.3 (bottled)
+{{< /text >}}
+
+If you find that the `bash-completion` package is _not_ installed, proceed with installing the `bash-completion` package with the following command:
+
+{{< text bash >}}
+$ brew install bash-completion
+{{< /text >}}
+
+Once the `bash-completion package` has been installed on your macOS system, also make sure to add the following line to your `~/.bash_profile` file:
+
+`[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"`
+
+{{< /tab >}}
+
+{{< tab name="Linux" cookie-value="linux" >}}
+
+If you are using a Linux-based operating system, you can install the Bash completion package with the `apt-get install bash-completion` command for Debian-based Linux distributions or `yum install bash-completion` for RPM-based Linux distributions, the two most common occurrences.
+
+Once the `bash-completion` package has been installed on your Linux system, also make sure to add the following line to your `~/.bash_profile` file:
+
+`[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"`
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Enabling `istioctl` auto-completion
+
+To enable `istioctl` completion on your system, follow the steps for your preferred shell:
+
+{{< tabset cookie-name="profile" >}}
+
+{{< tab name="Bash" cookie-value="bash" >}}
+
+### Installing the bash auto-completion file
```

Comment:
I don't think headings work inside tabs

Reply:
> I don't think headings work inside tabs

@frankbu mechanically or aesthetically?

Reply:
I thought both. Is it what is causing the linting errors?
```
  *  32733:30: ERROR: Unexpected end tag : li (line 32733)
  *  32958:30: ERROR: Unexpected end tag : li (line 32958)
```

Reply:
@frankbu ah! you could be right! Let me modify that and see if it clears up the linting issue....